src/queries/catalog/linter-query/linter-query-format.ts

```diff
@@ -27,6 +33,29 @@ export interface LinterQueryResult extends BaseQueryResult {
 	readonly results: { [L in LintingRuleNames]?: LintingResults<L>}
 }
 
+function rulesFromInput(rulesPart: string[]): (LintingRuleNames | ConfiguredLintingRule)[] {
+	return rulesPart.map(rule => {
+		const ruleName = rule.trim();
+		if(!(ruleName in LintingRules)) {
+			console.error(`Unknown linting rule '${ruleName}'`);
```

Comment:
could you please use the `output` passed by the repl and not just dump to the console? this output should also provide formatting options